Facepunch.Steamworks/Generated/SteamEnums.cs

```diff
@@ -546,7 +546,7 @@ internal enum ActivateGameOverlayToWebPageMode : int
 	//
 	// EPersonaChange
 	//
-	internal enum PersonaChange : int
+	public enum PersonaChange : int
```

Comment:
This needs to be updated in the generator too - not just here.

Reply:
I had overlooked this by mistake, corrections made as per my latest comment.